lib/rmg/ConnectionsPlacer.h

```diff
@@ -30,8 +30,7 @@ class ConnectionsPlacer: public Modificator
 	
 protected:
 	void collectNeighbourZones();
-	
-protected:
+
```

Comment:
I'd rather disable this rule.

It might be useful if clang-tidy is integrated in IDE & can show potential issues, but see no reason to enforce it since we sometimes use this to separate logical blocks

Reply:
restored, but my IDE is showing yellow marks)